app/models/deck.rb

```diff
@@ -0,0 +1,15 @@
+class Deck < ApplicationRecord
+    belongs_to :user
+    has_many :cards
+
+    validates :title, presence: true
+
+    scope :current, -> { where('current = ?', true) }

```

Comment:
current may be only one deck of user.
But for now, you can set 2 or more decks of User as a current. It is not ok.